### DISCUSSION TITLE
New Webmaster - Really simple questions here

### DISCUSSION THREAD

**Discussion by TSWrightSF** [code4recovery/12-step-meeting-list#1121] [Q&A] [answered]
I am a new webmaster for our local AA Central Office.  Our current website is at santafeaa.org.

We are creating a staging site in order to update the back-end of the existing site as well as making it easier for idiots like me to update and maintain.  On the recommendation of our outside web designer, the site is based on twenty-twenty-three Theme and is built using WP 6.2.  The thought is that the theme will probably be updated as our current Theme has not been updated in many years.

On the staging site, I have added the tsml plug in.  I have added a couple of dummy meetings.

Here are my stupid questions (which might want to be considered to be added to the FAQ or to the installation notes):

How do I get the meeting list to appear on the staged website?  I assume the address is mainurl/meetings/ (as noted in the import settings page of the plugin) but all I get there is a mal-formed page that ignores my site-wide header and footer, defaults to the stock theme and gives an error message of "Configuration error: data is not in the correct format."

Assuming that I could get the meeting list to appear, how do I format the meeting page so that it matches the style of the rest of the website?

How do I get the small box with upcoming meetings to appear on the side of our website pages?  Is there some code that has to be created in order to do that?

How do I get to the place where listed meetings can give us feedback to update or confirm their meeting listing?  Can that be done through the specific meeting page (preferable) or should I link to a google form to provide that info to us?

I apologize for the stupid questions, but the documentation seems to assume a fairly high level of CSS experience which I clearly don't have.

Thank you in advance -

Spence
Santa Fe, NM

**Comment by joshreisner** [comment]
Hi Spence, could you share your staging URL with us? Hard to know what's going on with the meetings page from the description. 

One thing I would try first is to go to Settings > Permalinks and use the postname structure, or this structure: <img width="647" alt="Screenshot 2023-04-29 at 10 14 41 AM" src="https://user-images.githubusercontent.com/1551689/235315261-e3c47fa9-77e8-45f5-bafb-2975c6ec5371.png"> (be sure to save changes)

If that doesn't help, then there may be something wrong with the WP setup, maybe there are some old files in the directory?

The upcoming meetings widget is available under Appearance > Widgets

You can enable a feedback form at Meetings > Settings (enter an email under Feedback Email Addresses)

If you can try those things and then provide a staging URL for us to see we can advise further, thanks

**Comment by TSWrightSF** [comment]
Thanks, Josh -

staging.santafeaa.org (obviously still a work in process)

I see Appearance > Widgets and it lists 4 types of widgets, but how do I
get the widget onto my page (see, I really am stupid).

Thank you for responding so quickly - and for all the work you have done on
this project.
--
Spencer

**Comment by joshreisner** [comment]
I'm sorry I'm not well-versed enough in the new WordPress block editor to know what it going on - I'm getting similar results to you when I switch to the Twenty Twenty Three theme. I will ask in our Slack to see if someone more knowledgable can assist you

**Reply by joshreisner** [reply]
I'm glad to see the basics of the meeting finder appear to be working though!

**Comment by TSWrightSF** [comment]
Maybe we should standardize on a different theme?

We were trying to be as "out of the box simple" as possible, thinking that
would be smarter for our situation.

But, you probably want the plugin to be somewhat theme-agnostic.

And the basics are working!
--
Spencer

**Comment by anchovie91471** [comment]
If you want to use the block editor to add the upcoming meetings, click the plus to add a block wherever you want in the page, choose “shortcode” and enter the [tsml_next_meetings] shortcode.

You can also add the meeting finder by doing the same thing and using [tsml_ui] as the shortcode.

**Comment by kiyote33** [comment]
I'd stay away from the 23 theme. It doesn't offer very much and requires to much work to make presentable. Tons of good simple themes out there including most of the WordPress themes prior to TwentyTwo. I personally like the free Astra theme where you can do something like this 
![image](https://user-images.githubusercontent.com/57125427/235353380-33789264-ca9c-4045-8fb8-5e349dd3aace.png)
, but the GeneratePress that we use on [Code For Recovery](https://code4recovery.org/) is a good example of a basic simple one.

**Comment by TSWrightSF** [comment]
Thanks, but that doesn't render anything on  the page.

Spencer

On Sat, Apr 29, 2023 at 6:39 PM Anthony Baggett ***@***.***>
wrote:

> If you want to use the block editor to add the upcoming meeting, click the
> plus to add a block wherever you want in the page, choose “shortcode” and
> enter the [tsml_next_meetings] shortcode.
>
> You can slaps add the meeting finder by doing the same thing and using
> [tsml_ui] as the shortcode.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/1121#discussioncomment-5764115>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4SFFKBU6VUNKEVVLQTQBZLXDWYE7ANCNFSM6AAAAAAXQKOCKE>
> .
> You are receiving this because you authored the thread.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/1121/comments/5764115
> @github.com>
>
-- 

--
Spencer

**Reply by anchovie91471** [reply]
Hmmm, that's strange. I have definitely used the shortcode block that way with 2023. Did you include the square brackets on either side like this? "[tsml_next_meetings]"

**Comment by TSWrightSF** [comment]
Thanks, Anthony -

I did include the square brackets.  Can't get it to appear.

Best -
--
Spencer

**Reply by anchovie91471** [reply]
That is very strange. Looking at your staging site, I only see 3 meetings on the main meetings page. Is that correct?

I’m wondering if they aren’t showing up because there isn’t a meeting in the system on that day. The upcoming meetings shortcode shows meetings for the current day of the week only and by default, shows the next 5, (though you can show more with extra shortcode parameters).

**Comment by TSWrightSF** [comment]
Hi Anthony -

Is there a list of shortcodes?   Still plugging away at this project.

Thanks -
--
Spencer

**Comment by TSWrightSF** [comment]
Are these them?

[tsml_meeting_count] = Number of meetings
[tsml_location_count] = Number of Locations
[tsml_group_count] = Number of Groups
[tsml_region_count] = Number of regions
[tsml_next_meetings count="5"] = Box that includes next 5 upcoming meetings

None of them seem to work - is it a WP 6.2 thing?

I note that the TSML suggests that the plugin is not tested with WP 6.2.

I've tried Block Themes as well as Classic themes.

Thanks, sorry to ask so many questions.
--
Spencer


On Mon, May 29, 2023 at 11:56 AM Spencer Wright ***@***.***> wrote:
>
> Hi Anthony -
>
> Is there a list of shortcodes?   Still plugging away at this project.
>
> Thanks -
> --
> Spencer
>
>
> On Sat, May 6, 2023 at 12:59 PM Anthony Baggett ***@***.***>
wrote:
>>
>> That is very strange. Looking at your staging site, I only see 3
meetings on the main meetings page. Is that correct?
>>
>> I’m wondering if they aren’t showing up because there isn’t a meeting in
the system on that day. The upcoming meetings shortcode shows meetings for
the current day of the week only and by default, shows the next 5, (though
you can show more with extra shortcode parameters).

**Reply by joshreisner** [reply]
they seem to all be working ok for me in 6.2. (i did notice a PHP error when including both the `tsml_region_count` and `tsml_region_list` shortcodes on the same page, so don't do that)

could you share a screenshot so we can see what's happening? doesn't sound like you've added the shortcodes to the page using a shortcode block

![admin](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/c2895afb-e1ea-49e9-a2d9-6d3e25e48826)
![page](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/62825d42-9412-4cfb-a647-4801b66c17b2)

**Comment by TSWrightSF** [comment]
Here is what I get as a page and as a post

Thanks -
--
Spencer

**Reply by SIA-WEBSUP** [reply]
@TSWrightSF , Did you mean to attach screenshots to the last reply?

what page has the upcoming meetings widget ??

**Reply by anchovie91471** [reply]
Have you tried using something like this in a theme template? (Like maybe index.php). 

`echo do_shortcode( '[tsml_meeting_count]' );`

This isn’t a permanent solution, but it seems like there might be a plug-in or something in your theme that is causing the shortcodes to fail.

Also, have you tried other shortcodes to see if it’s a general shortcode issue or if it’s specific to TSML?

**Comment by TSWrightSF** [comment, marked as answer]
Thank you all -

I have gotten the shortcodes to work thanks to your help.  I believe my
meetings database was screwed up as my dummy meetings were not assigned to
a group.  It looks like that option (the Individual meeting/Part of a group
radio buttons) messed me up when I entered the placeholder meetings.  Maybe
that could be better documented as it seems to trip people up.

The page with the upcoming meetings widget (I used shortcode rather than
the widget) is at staging.santafeaa.org

My "hello world" page to test the shortcodes is at
https://staging.santafeaa.org/hello-world/.  After importing our current
meeting list, it looks more like Josh's workup.

Would it make sense to do more of a project on documentation?  I'd be
willing to help out, even if I'm a total n00b on this.

Thank you, everyone.
--
Spencer
